code-challenges.js

```diff
@@ -16,6 +43,15 @@ var secretCodeWord2 = "gobbledygook"
 var arrayOfWords = ["Apple", "Banana", "Plum", "Cherry", "Kiwi", "Peach"]
 // Expected output: "Apple" "Banana" "Peach"
 
+//declare a function that takes in an array
+const arrayofAs = (arr, letter) => {
+    //filter through the array and look for words that contain a
+    let isA = arr.filter(letter => letter.includes('a'))
```

Comment:
this filter is iterating over ever word to check if that word includes "a", not every letter. renaming `letter` might give you a little more clarity. this also doesn't account for capital "A" like in "Apple"